python/lsst/ts/rubintv/background/historicaldata.py

```diff
@@ -166,15 +185,16 @@ async def store_events(self, events: list[Event], locname: str) -> None:
                 self._calendar[loc_cam][year][month][day] = seq_num
         logger.info("ending store_events")
 
-    async def storage_name_for_event(self, event: Event, locname: str) -> str:
+    async def storage_key_for_event(self, event: Event, locname: str) -> str:
```

Comment:
I think this function should not be asynchronous.

Reply:
Agreed!